Cloud Application Manager/Tutorials/ebcli-tutorial.md

```diff
@@ -0,0 +1,442 @@
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+
+### Overview
+
+This tutorial is meant to assist users of Cloud Application Manager to learn how to use [ebcli](https://pypi.python.org/pypi/ebcli) - our command line tool- to control their Cloud Application Manager account as follows:
+
+* For boxes - how to deploy, export, import as version or as a draft, etc using ebcli.
+* For instances - how to reconfigure, reinstall, shutdown, poweron, export, etc using ebcli.
+* For ECS images - how to manage them using ebcli.
+* For workspaces - how to list them using ebcli.
+
+ebcli calls Cloud Application Manager APIs in the backend to list, provision and manage the lifecycle of your workloads based on the box configuration.
+
+### Audience
+
+All Cloud Application Manager users who want to control their Cloud Application Manager account using ebcli - command line tool.
+
+
+### Prerequisites
+
+* Access to Cloud Application Manager [Site](https://cam.ctl.io/#/login) or to an on-premises CAM appliance.
+* The user must have an existing account to login into using username and password or his preferred authentication mechanism.
+
+
+## Installation of ebcli command line tool
+
+Before you start using ebcli, you need to install it using pip which is a package manager for Python packages, or modules if you like.
+
+If you don't have pip installed. Please follow this guide to install it into your computer: https://pip.pypa.io/en/stable/installing/
+
+**Terminal Command**
+
+Open a terminal in your computer and execute the installation command:
+
+```
+pip install ebcli
+```
+
+When the installation process ends pip should have installed ebcli and all of its required dependencies.
+
+
+### Login into your Cloud Application Manager account with ebcli
+
+Before you can control your Cloud Application Manager account with ebcli, you need to connect your account in Cloud Application Manager with ebcli.
+
+The ebcli needs a token to login into your Cloud Application Manager account.
+
+**Steps to create a token**
+
+* Open https://cam.ctl.io if you are using SaaS or https://YOUR_APPLIANCE_FQDN if you are using on premises of cam
+* Login with your username and password or your preferred authentication mechanism
+* Click on your user name menu (at top right corner of CAM user interface)
+* Click on 'Authentication Tokens'
+* Write a descriptive name for your token, i.e. ebcli
+* Click on 'Create Token'
+* Click on clipboard buttom at right of your token to copy it
+
+**Terminal Command**
+
+Open a terminal in your computer and execute the login command:
+
+```
+ebcli login
+```
+
+You will be prompt for introduccing your 'Authentication Token:', just paste your token, and hit Enter.
+
+Now your computer will ask your keyring password to store your Cloud Application Manager token safely in your keyring.
```

Comment:
Hmmm, does this work the same in Windows?

Reply:
windows? WHAT is windows? ... I believe it works similarly. But I'll test it.